template/Dockerfile.blade.php

```diff
@@ -84,6 +84,10 @@
     && apk del .build-deps \
     && rm -rf /var/cache/apk/* /tmp/*
 
+@if ($with_node ?? false)
+RUN apk add --update --no-cache npm
```

Comment:
i would add yarn too... and i would build this from normal php, not from scratch